Source/WTF/wtf/Compiler.h

```diff
@@ -393,7 +393,7 @@
 
 /* REFERENCED_FROM_ASM */
 
-#if !defined(REFERENCED_FROM_ASM) && COMPILER(GCC_COMPATIBLE)
+#if !defined(REFERENCED_FROM_ASM) && (COMPILER(GCC) || COMPILER(CLANG))
```

Comment:
We need one more change for https://bugs.webkit.org/show_bug.cgi?id=260689 .
We have to consider enabling WTF_COMPILER_GCC_COMPATIBLE .

Reply:
I tried enabling WTF_COMPILER_GCC_COMPATIBLE locally.
I learned that it's not very easy. Never mind WTF_COMPILER_GCC_COMPATIBLE at the moment.